test/groovy/org/ods/usecase/LeVADocumentUseCaseSpec.groovy

```diff
@@ -820,16 +820,25 @@ class LeVADocumentUseCaseSpec extends SpecHelper {
         then:
         def e = thrown(RuntimeException)
         e.message == "Error: Jira query returned 0 issues: '${jqlQuery}'."
+    }
+
+    def "notify LeVA document issue with query returning more than 1 issue"() {
+        given:
+        jiraUseCase = Spy(new JiraUseCase(project, steps, util, Mock(JiraService)))
+        usecase = Spy(new LeVADocumentUseCase(project, steps, util, docGen, jenkins, jiraUseCase, levaFiles, nexus, os, pdf, sq))
+
+        def documentType = "myType"
+        def message = "myMessage"
+
+        def jqlQuery = [jql: "project = ${project.key} AND issuetype = 'LeVA Documentation' AND labels IN (LeVA_Doc:${documentType})"]
+        def documentIssues = createJiraDocumentIssues()
 
         when:
         usecase.notifyJiraTrackingIssue(documentType, message)
 
         then:
         1 * jiraUseCase.jira.getIssuesForJQLQuery(jqlQuery) >> documentIssues
-
-        then:
-        e = thrown(RuntimeException)
```

Comment:
@hrcornejo We should test the exception being thrown if there are 0 issues

Reply:
That is done in the other method